apps/spruce/src/constants/externalResources.ts

```diff
@@ -35,6 +35,9 @@ export const githubChecksAliasesDocumentationUrl = `${projectDistroSettingsDocum
 export const githubPermissionsDocumentationUrl =
   "https://docs.github.com/en/rest/apps/apps#create-an-installation-access-token-for-an-app";
 
+export const githubTokenPermissionRestrictionsUrl =
+  "TODO: Documentation must be written by app team. Update this URL in DEVPROD-8967.";
```

Comment:
[nit] Slight preference to include this as a comment and set the value to an empty string so that there's no chance of setting an invalid href